lib/matplotlib/widgets.py

```diff
@@ -3142,8 +3199,11 @@ class PolygonSelector(_SelectorWidget):
     point.
     """
 
+    @_api.rename_parameter("3.5", "lineprops", "props")
+    @_api.rename_parameter("3.5", "markerprops", "handle_props")
+    @_api.rename_parameter("3.5", "vertex_select_radius", "grab_range")
     def __init__(self, ax, onselect, useblit=False,
-                 lineprops=None, markerprops=None, vertex_select_radius=15):
+                 props=None, handle_props=None, grab_range=10):
```

Comment:
Why did the default change from 15?

Reply:
This is to be consistent with the default value of the other selectors.